ui/.env.goerli

```diff
@@ -1,13 +1,13 @@
-NEXT_PUBLIC_NATION_ADDRESS="0xE73B90231b9301a47556c246E7271769A30BDdEF"
-NEXT_PUBLIC_VENATION_ADDRESS="0xF7deF1D2FBDA6B74beE7452fdf7894Da9201065d"
+NEXT_PUBLIC_NATION_ADDRESS="0x333A4823466879eeF910A04D473505da62142069"
+NEXT_PUBLIC_VENATION_ADDRESS="0xf7def1d2fbda6b74bee7452fdf7894da9201065d"
 NEXT_PUBLIC_VENATION_REQUIRED_STAKE=2
 NEXT_PUBLIC_BALANCER_VAULT_ADDRESS="0xBA12222222228d8Ba445958a75a0704d566BF2C8"
 NEXT_PUBLIC_BALANCER_NATION_ETH_POOL_ID="0x0bf37157d30dfe6f56757dcadff01aed83b08cd600020000000000000000019a"
```

Comment:
@gregfromstl @luisivan Do you have the `NEXT_PUBLIC_BALANCER_VAULT_ADDRESS` and `NEXT_PUBLIC_BALANCER_NATION_ETH_POOL_ID` for the Goerli test network?

I got this error when launching the app:

<img width="783" alt="Screen Shot 2022-05-21 at 11 19 12 AM" src="https://user-images.githubusercontent.com/95955389/169633906-ffafdece-b598-4f9f-a865-9c57aeeb791b.png">

```
next-dev.js?3515:32 Error: call revert exception; VM Exception while processing transaction: reverted with reason string "BAL#500" [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method="getPoolTokens(bytes32)", data="0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000742414c2335303000000000000000000000000000000000000000000000000000", errorArgs=["BAL#500"], errorName="Error", errorSignature="Error(string)", reason="BAL#500", code=CALL_EXCEPTION, version=abi/5.6.2)
    at Logger.makeError (index.js?dd68:219:1)
    at Logger.throwError (index.js?dd68:228:1)
    at Interface.decodeFunctionResult (interface.js?1df7:345:1)
    at eval (index.js?b70c:293:1)
    at Generator.next (<anonymous>)
    at fulfilled (index.js?b70c:5:43)
```

And this is probably because of a wrong value in one of these variables?

Reply:
@aahna-ashina we currently don't mock the Balancer pool on Goerli, nor we have one. We could create one, if Balancer maintains a Goerli deployment